### PR TITLE
[PRTL-3244] outer dendrogram line enhancements

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

PRTL-3244

## Environment to be used in testing

- [ ] `prod`
- [ ] `dev-oicr`
- [x] `qa-purple` for gene ex data

## Description of Changes

- determine outer dendrogram line more reliably
- remove click events and cursor
- change text of tooltips
- handle situations like clicking on inner dendrogram to select it -> then clicking on outer dendrogram to de-select the first one
- reposition tooltips for row dendrograms

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js [hunk 1]
```diff
@@ -2642,38 +2650,45 @@ import { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
     });
   };
 
-  InCHlib.prototype._highlight_cluster = function (path_id) {
+  InCHlib.prototype._highlight_cluster = function (path_id, id) {
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js:2653]: What's the difference between those two ids?

File: src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js [hunk 2]
```diff
@@ -1589,8 +1589,7 @@ import { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
       const y1 = self._get_y1(node_neighbourhood, current_left_count, current_right_count);
       let y2 = self._get_y2(node_neighbourhood, current_left_count, current_right_count);
       let x1 = self._hack_round(self.distance - self.distance_step * node.distance);
-      x1 = (x1 == 0) ? 2 : x1;
-
+      x1 = (x1 === 0) ? 2 : x1;
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js:1592]: 🙌

File: src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js (hunk 1 above)
Comment (samrichca) [reply src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js:2653]: the `path_id` is used to construct the `id` and doesn't contain all of the same information - in this case it doesn't contain `_top_node` for the top dendrogram node

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js:2653]: meaning, the `path_id` is sort of the original, and then the id would be the one actually used in the DOM.
is that correct?

Comment (samrichca) [reply src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js:2653]: something like that. i figured it was safer to customize the constructed, final ID than its constituent parts.

**Review by caravinci — APPROVED** [review]
Squasharoooo 🦘

**Pull request merged**